newspaper/cleaners.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, config):
             .create("\n", "\n\n")\
             .append("\t")\
             .append("^\\s+$")
+        self.contains_article = './/article|.//*[@id="article"]|.//*[@itemprop="articleBody"]'
```

Comment:
nit: rename this to be `contains_article__re`? similar to the other variable names?